src/main/java/org/sonarlint/intellij/ui/ReportPanel.java

```diff
@@ -171,7 +171,7 @@ private void updateOnSelect(LiveFinding liveFinding) {
     var moduleForFile = ProjectRootManager.getInstance(project).getFileIndex().getModuleForFile(liveFinding.psiFile().getVirtualFile());
     rulePanel.setRuleKey(moduleForFile, liveFinding.getRuleKey(), null);
     SonarLintUtils.getService(project, EditorDecorator.class).highlightFinding(liveFinding);
-    flowsTree.getEmptyText().setText("No finding selected");
+    flowsTree.getEmptyText().setText("Selected finding doesn't have flows");
```

Comment:
What if no findings are selected?

Reply:
By default, it is initialized as `No finding selected`
If the user clicks on a non `SecurityHotspotNode` or `IssueNode` (meaning root node or file node) it will be cleared as `No finding selected` as well